client/src/components/JurisdictionAdmin/RoundManagement.test.tsx

```diff
@@ -165,20 +195,101 @@ describe('RoundManagement', () => {
       apiCalls.getSettings(auditSettings.batchComparisonAll),
       jaApiCalls.getUser,
       apiCalls.getBatches(batchesMocks.emptyInitial),
-      apiCalls.getJAContests({ contests: contestMocks.oneTargeted }),
       apiCalls.getBatches(batchesMocks.emptyInitial),
+      apiCalls.getJAContests({ contests: contestMocks.oneTargeted }),
+      apiCalls.getJAContests({ contests: contestMocks.oneTargeted }),
     ]
+    ;(BlobProvider as jest.Mock).mockImplementation(
+      ({ children, document }: ReactPDF.BlobProviderProps) => {
+        return (
+          <>
+            {children({
+              blob: null,
+              error: null,
+              loading: false,
+              url: 'blob:http://blob-url',
+            })}
+            {/* BlobProvider doesn't normally render the document in the DOM, but we do so here to
+              facilitate testing */}
+            <div data-testid="pdf">{document}</div>
+          </>
+        )
+      }
+    )
+
     await withMockFetch(expectedCalls, async () => {
       renderView({
-        round: roundMocks.incomplete,
         auditBoards: auditBoardMocks.unfinished,
         createAuditBoards: jest.fn(),
+        round: roundMocks.incomplete,
+      })
+      await screen.findByRole('button', {
+        name: /Download Aggregated Batch Retrieval List/,
```

Comment:
I know we discussed removing the "Aggregated" in this copy. Taking care of that in a separate PR!